googler

```diff
@@ -334,6 +334,15 @@ except GetoptError as e:
     print("googler:", e)
     sys.exit(1)
 
+# Set up logging
```

Comment:
Well, kind of... You can wrap the whole logging block in `if debug` if you really want it that way. But then you lose the advantage of `logging`. Is there a particular reason (the other common is rather vague) you want to leave `if debug` everywhere? Because when `debug` is not set, `logger` retains its default level, that is `logging.NOTSET`, so when we subsequently call `logger.debug` the messages are processed by the root logger, the default level of which is `logging.WARNING`, so the debug messages are automatically ignored. On the other hand, if `debug` is set, then `logger`'s level is set to `logging.DEBUG` such that all subsequent `logger.debug` messages go through. So basically we have an implicit `if debug` around each `logger.debug` call already. This is the beauty of `logging`, and still dropping `if debug` everywhere kind of defeats the purpose, then there's little benefit of using `logging` over just `print`. If you don't like the idea of passing to root logger, we can be more explicit and do

```python
if debug:
    logger.setLevel(logger.DEBUG)
else:
    logger.setLevel(logger.WARNING)
```